### DISCUSSION TITLE
i cant get multiline tips anymore! only one by one!

### DISCUSSION THREAD

**Discussion by xuxin2019-f** [community/community#15038] [Copilot Conversations]
Starting today I can't get multiline tips from copilot, only one at once! it means i cant see the 10 solutions. why

**Comment by AAAAAXfda-zz** [comment]
vouch

**Reply by xuxin2019-f** [reply]
what do you mean

**Reply by AAAAAXfda-zz** [reply]
I'm confirming what you said, I'm getting the same issue aswell
